dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/config_chart_options.tsx

```diff
@@ -92,6 +93,15 @@ export const ConfigChartOptions = ({
             vizState,
             ...schema.props,
           };
+        } else if (schema.eleType === 'slider') {
+          params = {
```

Comment:
check if default state of param is added

Reply:
Default state for params is already added in the else condition.